lib/scenic/statements.rb

```diff
@@ -51,7 +51,7 @@ def create_view(name, version: 1, sql_definition: nil, materialized: false)
     # @example Drop a view, rolling back to version 3 on rollback
     #   drop_view(:users_who_recently_logged_in, revert_to_version: 3)
     #
-    def drop_view(name, revert_to_version: nil, materialized: false)
+    def drop_view(name, revert_to_version: nil, materialized: false, sql_definition: nil)
```

Comment:
Unused method argument - `revert_to_version`.<br>Unused method argument - `sql_definition`.<br>Line is too long. [89/80]